src/test/java/com/spring/guideance/service/TagServiceTest.java

```diff
@@ -0,0 +1,203 @@
+package com.spring.guideance.service;
+
+import com.spring.guideance.post.domain.Article;
+import com.spring.guideance.post.dto.request.CreateArticleDto;
+import com.spring.guideance.post.repository.ArticleRepository;
+import com.spring.guideance.tag.domain.ArticleTag;
+import com.spring.guideance.tag.domain.Tag;
+import com.spring.guideance.tag.dto.ResponseTagDto;
+import com.spring.guideance.tag.repository.ArticleTagRepository;
+import com.spring.guideance.tag.repository.TagRepository;
+import com.spring.guideance.tag.repository.UserTagRepository;
+import com.spring.guideance.tag.service.TagService;
+import com.spring.guideance.user.domain.User;
+import com.spring.guideance.user.dto.request.CreateUserDto;
+import com.spring.guideance.user.repository.UserRepository;
+import com.spring.guideance.util.exception.TagException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@Transactional
+public class TagServiceTest {
+
+    @Autowired
+    TagService tagService;
+    @Autowired
+    UserTagRepository userTagRepository;
+    @Autowired
+    ArticleTagRepository articleTagRepository;
+    @Autowired
+    TagRepository tagRepository;
+    @Autowired
+    UserRepository userRepository;
+    @Autowired
+    ArticleRepository articleRepository;
+
+    // 태그 생성
+    @Test
+    public void 태그생성(){
+        // given
+        String tagName = "tag1";
+
+        // when
+        Long tagId = tagService.createTag(tagName);
+
+        // then
+        assertEquals(tagName, tagRepository.findById(tagId).get().getTagName());
```

Comment:
expected와 actual이라는 네이밍은 어떻게 생각하시나요?
```suggestions
String actual = tagRepository.findById(tagId).get().getTagName();
assertEquals(expected, actual);
```

Reply:
테스트 코드 네이밍이 전반적으로 소홀했던 것 같아, 한번 리팩토링을 진행할 계획입니다!